src/app/environment.py

```diff
@@ -28,6 +28,9 @@ class Settings(BaseSettings):
     # Logging
     log_level: str = "info"
 
+    # Sentry
+    sentry_dsn: str = ""
```

Comment:
This might be a good usecase for a [URL type provided by Pydantic](https://pydantic-docs.helpmanual.io/usage/types/#urls). Also, I think it could be `Optional[<url type>]` since that's what the [Sentry client accepts](https://github.com/getsentry/sentry-python/blob/de0bc5019c715ecbb2409a852037530f36255d75/sentry_sdk/consts.py#L49).